src/dtls.c

```diff
@@ -907,7 +938,13 @@ int DoClientHelloStateless(WOLFSSL* ssl, const byte* input,
 #endif
 
     if (ch.cookie.size == 0 && ch.cookieExt.size == 0) {
-        ret = SendStatelessReply((WOLFSSL*)ssl, &ch, isTls13);
+#ifdef WOLFSSL_DTLS_CH_FRAG
+        /* Don't send anything here when processing fragment */
+        if (isFirstCHFrag)
+            ret = BUFFER_ERROR;
```

Comment:
If we error out on `isFirstCHFrag` without `WOLFSSL_DTLS_CH_FRAG` then we don't need this code it can be compiled out. Can you give a fuller example of what you had in mind?